ranger-zookeeper/src/main/java/com/flipkart/ranger/zookeeper/serviceprovider/ZkNodeDataSink.java

```diff
@@ -1,9 +1,25 @@
-package com.flipkart.ranger.zookeeper.zk;
+/*
+ * Copyright 2015 Flipkart Internet Pvt. Ltd.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.flipkart.ranger.zookeeper.serviceprovider;
 
 import com.flipkart.ranger.core.model.NodeDataSink;
 import com.flipkart.ranger.core.model.Service;
 import com.flipkart.ranger.core.model.ServiceNode;
 import com.flipkart.ranger.core.util.Exceptions;
+import com.flipkart.ranger.zookeeper.common.ZkNodeDataStoreConnector;
```

Comment:
No way to comment below, use lombok in the few places it is missing

Reply:
Part of the lombok cleanup